bindgen/src/output-directory.ts

```diff
@@ -62,7 +64,7 @@ export function createOutputDirectory(outputPath: string): OutputDirectory {
   }
   const openFiles: OutputFile[] = [];
   return {
-    file(filePath: string, ...formatters: FormatterName[]) {
+    file(filePath: string, ...formatters: Formatter[]) {
```

Comment:
For now, we could consider changing this to only take at most one `formater?: Formatter` argument, which avoids the whole defined order problem. I somewhat doubt that we will need to support multiple formatters in the future, if other SDKs stick to the pattern of doing the checking as part of the build system rather than the generator.

Reply:
I'm okay with that 👍